lib/install/management/delete.go

```diff
@@ -51,12 +52,15 @@ func (d *Dispatcher) DeleteVCH(conf *config.VirtualContainerHostConfigSpec, cont
 	var err error
 	var vmm *vm.VirtualMachine
 
+	// TODO: we previously found this appliance...there's no need to
```

Comment:
Before merging to `master`, I think it would be good to either resolve or file issues for all TODOs.

Reply:
Removed...